docs/notifications.md

```diff
@@ -20,11 +20,11 @@ GCM only wakes up the app. The app syncs with the user's homeserver to get the m
 ```
 Homeserver ----> Sygnal ----> GCM ----> Riot
                                         (Sync) ----> Homeserver
-                                               <---- 
+                                               <----
```

Comment:
Why this file has been modified? Please revert

Reply:
There are no modifications.  Accidently it got deleted.  I re-add the docs folder upstream riot-android repo.

Reply:
only white spaces have removed